plugin/path_role.go

```diff
@@ -37,6 +38,54 @@ const (
 	maxJwtExpMaxMinutes int = 60
 )
 
+var (
+	// The default gce_alias is "instance_id". The default fields
```

Comment:
For 1.4.1 I think we should choose defaults that match the 1.4 set since people will not be expecting that to change. AWS is somewhat different since that metadata was introduced quite recently and has known issues in some environments.

Reply:
Sure, I can do that! Incorporating....

Reply:
I'll add a task for 1.5 to update these.

Reply:
https://github.com/hashicorp/vault-plugin-auth-gcp/pull/90/commits/8d153f0a6e713a24d6abf55ac58777a53bcad734